packages/server/src/api/endpoints/payment.ts

```diff
@@ -15,16 +15,36 @@ export function paymentEndpoints(server: FastifyInstance): void {
 
       const project = await database.projects.findOne(params?.projectId);
```

Comment:
@mariusheine If this line would throw an error, we would get a log line like:

```
[14:52:42 UTC] ERROR: select "c0".* from "customer" as "c0" where "c0"."_id" = '113' and "c0"."project__id" = 'aca1c2a8-6550-4c22-a197-59a60dec8f0c' limit 1 - invalid input syntax for type uuid: "113"
 reqId: "req-e"
    req: {
      "method": "POST",
      "url": "/subscription",
      "hostname": "localhost:7171",
      "remoteAddress": "127.0.0.1",
      "remotePort": 58506
    }
    res: {
      "statusCode": 500
    }
    err: {
      "type": "DriverException",
      "message": "select \"c0\".* from \"customer\" as \"c0\" where \"c0\".\"_id\" = '113' and \"c0\".\"project__id\" = 'aca1c2a8-6550-4c22-a197-59a60dec8f0c' limit 1 - invalid input syntax for type uuid: \"113\"",
      "stack":
          /home/anton/Projects/geprog/gringotts/node_modules/.pnpm/@mikro-orm+core@5.6.11_@mikro-orm+migrations@5.6.11_@mikro-orm+postgresql@5.6.11/node_modules/@mikro-orm/core/platforms/ExceptionConverter.js:8
                  return new exceptions_1.DriverException(exception);
                         ^
          
          DriverException: select "c0".* from "customer" as "c0" where "c0"."_id" = '113' and "c0"."project__id" = 'aca1c2a8-6550-4c22-a197-59a60dec8f0c' limit 1 - invalid input syntax for type uuid: "113"
              at PostgreSqlExceptionConverter.convertException (/home/anton/Projects/geprog/gringotts/node_modules/.pnpm/@mikro-orm+core@5.6.11_@mikro-orm+migrations@5.6.11_@mikro-orm+postgresql@5.6.11/node_modules/@mikro-orm/core/platforms/ExceptionConverter.js:8:16)
              at PostgreSqlExceptionConverter.convertException (/home/anton/Projects/geprog/gringotts/node_modules/.pnpm/@mikro-orm+postgresql@5.6.11_@mikro-orm+core@5.6.11_@mikro-orm+migrations@5.6.11/node_modules/@mikro-orm/postgresql/PostgreSqlExceptionConverter.js:42:22)
              at PostgreSqlDriver.convertException (/home/anton/Projects/geprog/gringotts/node_modules/.pnpm/@mikro-orm+core@5.6.11_@mikro-orm+migrations@5.6.11_@mikro-orm+postgresql@5.6.11/node_modules/@mikro-orm/core/drivers/DatabaseDriver.js:197:54)
              at null.<anonymous> (/home/anton/Projects/geprog/gringotts/node_modules/.pnpm/@mikro-orm+core@5.6.11_@mikro-orm+migrations@5.6.11_@mikro-orm+postgresql@5.6.11/node_modules/@mikro-orm/core/drivers/DatabaseDriver.js:201:24)
              at processTicksAndRejections (node:internal/process/task_queues:96:5)
              at PostgreSqlDriver.find (/home/anton/Projects/geprog/gringotts/node_modules/.pnpm/@mikro-orm+knex@5.6.11_@mikro-orm+core@5.6.11_@mikro-orm+migrations@5.6.11_pg@8.9.0/node_modules/@mikro-orm/knex/AbstractSqlDriver.js:49:24)
              at PostgreSqlDriver.findOne (/home/anton/Projects/geprog/gringotts/node_modules/.pnpm/@mikro-orm+knex@5.6.11_@mikro-orm+core@5.6.11_@mikro-orm+migrations@5.6.11_pg@8.9.0/node_modules/@mikro-orm/knex/AbstractSqlDriver.js:67:21)
              at SqlEntityManager.findOne (/home/anton/Projects/geprog/gringotts/node_modules/.pnpm/@mikro-orm+core@5.6.11_@mikro-orm+migrations@5.6.11_@mikro-orm+postgresql@5.6.11/node_modules/@mikro-orm/core/EntityManager.js:329:22)
              at SqlEntityManager.findOneOrFail (/home/anton/Projects/geprog/gringotts/node_modules/.pnpm/@mikro-orm+core@5.6.11_@mikro-orm+migrations@5.6.11_@mikro-orm+postgresql@5.6.11/node_modules/@mikro-orm/core/EntityManager.js:363:22)
              at Object.handler (/home/anton/Projects/geprog/gringotts/packages/server/src/api/endpoints/subscription.ts:61:24)
          
          previous /home/anton/Projects/geprog/gringotts/node_modules/.pnpm/pg-protocol@1.6.0/node_modules/pg-protocol/dist/parser.js:287
                  const message = name === 'notice' ? new messages_1.NoticeMessage(length, messageValue) : new messages_1.DatabaseError(messageValue, length, name);
                                                                                                           ^
          
          error: select "c0".* from "customer" as "c0" where "c0"."_id" = '113' and "c0"."project__id" = 'aca1c2a8-6550-4c22-a197-59a60dec8f0c' limit 1 - invalid input syntax for type uuid: "113"
              at Parser.parseErrorMessage (/home/anton/Projects/geprog/gringotts/node_modules/.pnpm/pg-protocol@1.6.0/node_modules/pg-protocol/dist/parser.js:287:98)
              at Parser.handlePacket (/home/anton/Projects/geprog/gringotts/node_modules/.pnpm/pg-protocol@1.6.0/node_modules/pg-protocol/dist/parser.js:126:29)
              at Parser.parse (/home/anton/Projects/geprog/gringotts/node_modules/.pnpm/pg-protocol@1.6.0/node_modules/pg-protocol/dist/parser.js:39:38)
              at Socket.<anonymous> (/home/anton/Projects/geprog/gringotts/node_modules/.pnpm/pg-protocol@1.6.0/node_modules/pg-protocol/dist/index.js:11:42)
              at Socket.emit (node:events:527:28)
              at addChunk (node:internal/streams/readable:315:12)
              at readableAddChunk (node:internal/streams/readable:289:9)
              at Socket.Readable.push (node:internal/streams/readable:228:10)
              at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
      "length": 102,
      "name": "DriverException",
      "severity": "ERROR",
      "code": "22P02",
      "position": "58",
      "file": "uuid.c",
      "line": "134",
      "routine": "string_to_uuid"
    }
```

Reply:
@anbraten Alright, that looks fine/sufficient.